primer/primitives
# Deprecate marketing button tokens

## Summary

Part of https://github.com/github/primer/issues/1366
Closes https://github.com/github/primer/issues/1487


## List of notable changes:



- **deprecated**  design token for "mktg.btn" because we have replacements in `@primer/brand-primitives`
- **removed** new marketing tokens from v2 property files, because they won't be used at all and can be removed immediately.


```diff
- ---color-mktg-btn-bg
+ --brand-Button-background-base 
```

```diff
- --color-mktg-btn-shadow-outline
+ --brand-Button-shadow-default
```

```diff
- --color-mktg-btn-shadow-focus
+ --brand-Button-shadow-focus
```

```diff
- --color-mktg-btn-shadow-hover
+ --brand-Button-shadow-primary-hover
```

```diff
- --color-mktg-btn-shadow-muted
+ --color-mktg-btn-shadow-hover-muted
```

## What should reviewers focus on?

- Have I completed the deprecation step correctly? And did I miss any steps? 
- Do we deprecate in a minor? Or go straight to removal with a major? 

## Steps to test:

1. Check the package output to verify tokens appear in deprecated.json (via Upkg. E.g. on [canary release](https://unpkg.com/browse/@primer/primitives@0.0.0-20221230115427/dist/deprecated/colors.json))
1. Check deprecations appear in docs
1. Check the replacement tokens cover all use-cases 

## Supporting resources (related issues, external links, etc):

- [Primer Brand](https://github.com/primer/brand)

## Contributor checklist:


## Reviewer checklist:
